bench-tps/src/cli.rs

```diff
@@ -324,22 +325,32 @@ pub fn build_args<'a>(version: &'_ str) -> App<'a, '_> {
             Arg::with_name("tpu_client")
                 .long("use-tpu-client")
                 .conflicts_with("rpc_client")
+                .conflicts_with("thin_client")
                 .takes_value(false)
                 .help("Submit transactions with a TpuClient")
         )
+        .arg(
+            Arg::with_name("thin_client")
+                .long("use-thin-client")
+                .conflicts_with("rpc_client")
+                .conflicts_with("tpu_client")
+                .takes_value(false)
+                .hidden(hidden_unless_forced())
+                .help("Submit transactions with a ThinClient")
```

Comment:
Maybe add a note here that usage of ThinClient is discouraged and that ThinClient will be deprecated?